server/src/DBTablesMonitoring.cc

```diff
@@ -2540,6 +2540,23 @@ HatoholError DBTablesMonitoring::getNumberOfMonitoredItemsPerSecond
 	return HatoholError(HTERR_OK);
 }
 
+size_t DBTablesMonitoring::getNumberOfEvents(const EventsQueryOption &option)
+{
+	DBClientJoinBuilder builder(tableProfileEvents, &option);
+	string stmt =
```

Comment:
Just impression: Don't take care.
We want use 'constexpr' for this kind of lines.
As a result, SQL statements are generated on the build time, instead of runtime.